api/src/routers/reviews.js

```diff
@@ -0,0 +1,78 @@
+import express from "express";
+import knex from "../database_client.js";
+const router = express.Router();
+
+// router to get all the reviews
+router.get("/", async (req, res, next) => {
+  try {
+    const reviews = await knex("review");
+    res.json(reviews);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// router to get a specific review
+router.get("/meal/:meal_id", async (req, res, next) => {
+  const { meal_id } = req.params;
+  try {
+    const reviews = await knex("review").where({ meal_id });
+    res.json(reviews);
+  } catch (error) {
+    next(error);
+  }
+});
```

Comment:
you are missing some logic here :) Also remember to handle the case where the meal simply isnt found. Hint: Use error code 404 for for not found!

Reply:
Also what is the meal doing here? :smile:

Reply:
meal id is used to filter reviews related to a specific dish. its looking for all reviews for this specific meal id